src/diamond/server.py

```diff
@@ -93,8 +93,8 @@ def run(self):
             sys.exit(1)
 
         handlers = self.config['server'].get('handlers')
-        handlers = handlers.split(',')
-        handlers = map(str.strip, handlers)
+        if isinstance(handlers, basestring):
+            handlers = [handlers]
```

Comment:
There is no need. If there is a comma, indicating multiple elements, a list will be returned with each of the elements. The comma is thus automatically omitted.